Dockerfile

```diff
@@ -27,10 +24,7 @@ RUN yarn install --frozen-lockfile
 # Copy all files to /app (except what is defined in .dockerignore)
 COPY . /app/
 
-# Compile assets and run webpack
-# Run in rails test environment to avoid loading development gems
-RUN RAILS_ENV=test bundle exec rails assets:precompile
-ENV RAILS_SERVE_STATIC_FILES=true
```

Comment:
I guess this is not something not related to the ruby update.
It's fine anyway, maybe just put a comment to show why we don't need them.